tasks/header.py

```diff
@@ -96,7 +97,7 @@ def update_header(file: str, header: t.List, header_position: t.Dict):
     header = "".join(header)
 
     # reinsert the header on the given position
-    content[header_position["starting_pos"]:header_position["ending_pos"]] = header
+    content.insert(header_position["starting_pos"], header)
```

Comment:
@ptnapoleon this is a rollback of a change introduced [here](https://github.com/DataDog/chaos-controller/commit/b395eb7b9066423c8b34066e32d7dc35eb614cc3)

Could you confirm it's OK to rollback this?

I faced an issue using `make generate-mock` that leads to the go `package` being removed 🙈 

Why did you had to change this in the first place?

Reply:
I made this change because I was having an issue where the new header was being inserted in front of the old header, rather than an overwrite. I tried changing some headers to be wrong and running `make header-check` with your branch here, and i'm seeing the right behavior. So.... seems fine?

Reply:
Alright, i looked back at the 2021 and 2022 overwrites, and it seems I should have just used a "find all + replace" in GoLand, rather than trying to change header.py to overwrite headers, when it only seems capable of inserting headers into files that lack them. I think your changes here are correct and sufficient